packages/web/src/scss/components/Tooltip/README.md

```diff
@@ -224,14 +224,49 @@ on the `.Tooltip__arrow` to control it and prevent conflicts with the default CS
 </div>
 ```
 
-### Example
+### Floating UI functionality
 
-💻 Check our [example] that uses external library
-[Floating UI][floating-ui] (see the [JS source](./floating-ui-example.mjs)).
+To enable the Floating UI functionality, you need to activate the JS plugin, wrap your tooltip with an element having the `data-spirit-element="tooltip-wrapper"` data attribute, and add the `data-spirit-placement-controlled` attribute to your tooltip element.
+
+```html
+<div class="spirit-feature-tooltip-enable-data-placement">
+  <div class="TooltipWrapper d-inline-block" data-spirit-element="tooltip-wrapper">
+    <button type="button" id="tooltip-trigger" data-spirit-toggle="tooltip" data-spirit-target="#floating-ui-example">
+      Toggle tooltip
+    </button>
+    <div
+      id="floating-ui-example"
+      class="Tooltip is-hidden"
+      data-spirit-placement="bottom"
+      data-spirit-flip="true"
+      data-spirit-flip-cross-axis="true"
+      data-spirit-flip-fallback-placements="bottom, right, top"
+      data-spirit-placement-controlled
+    >
+      Hello there!
+      <span class="Tooltip__arrow" data-spirit-element="arrow"></span>
+    </div>
+  </div>
+</div>
+```
```

Comment:
This is cool, I'd just add a very basic example before this one, showing just the things that are _essential_ to launch a Floating-UI tooltip.

Reply:
I left only a basic example there because the example with more options is almost the same, just with additional data attributes listed in the table below.